stroeer/fragment/stage/v1/stage_service.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+import "stroeer/page/section/v1/section_page.proto";
+
+package stroeer.fragment.stage.v1;
+
+option go_package = "github.com/stroeer/tapir/go/stroeer/fragment/stage/v1;stage";
+option java_multiple_files = true;
+option java_package = "de.stroeer.fragment.stage.v1";
+
+service StageService {
+  /**
+    ############################################################################
+    # Description
+    ############################################################################
+    Get single stages by requesting them via agreed tokens, e.g. "schlagzeilen" or "meistgelesen".
```

Comment:
I find the mix of calling the identifiers `id` and/or `token` in the comments confusing. I'd go with id/identifier

Reply:
done